packages/mobile/components/StrongholdUnlock.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="typescript">
+    import { localize } from '@core/i18n'
+    import { unlockStronghold } from '@core/profile'
+    import { Button, PasswordInput, Text, HTMLButtonType } from 'shared/components'
+
+    export let subtitle: string | undefined = undefined
+    export let busyMessage: string = ''
+
+    export let onSuccess: (..._: any[]) => Promise<unknown>
+    export let onCancel: () => unknown
+
+    let password: string
+    let error: string
+    let isBusy = false
```

Comment:
This `isBusy` is somehow not working 100% because while it was loading I could cancel the password drawer which lead to a weird behavior
![image](https://user-images.githubusercontent.com/3624944/198013789-973019f5-6933-4318-a614-b7d4b8d07410.png)
if you click cancel at this point it goes to the previous view and when it finishes creating the wallet in the background the drawer closes by itself, we should block that cancel button imo